notebooks/en/semantic_reranking_elasticsearch.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>in this example there's a lot of redirections mid-reading, it will undermine SEO of this page, and break readability, maybe put it at the end of the tutorial instead? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>